### DISCUSSION TITLE
How to put  navigation tabs inside of the header?

### DISCUSSION THREAD

**Discussion by jcyrss** [squidfunk/mkdocs-material#6789] [Q&A] [answered]
I leave only icon and site name on the header, so the whole header looks emtpy. It would be nice to put  navigation tabs inside of the header instead of below the header.

Just like mkdocs built-in theme does

![image](https://github.com/squidfunk/mkdocs-material/assets/10496014/020e551d-8078-46fe-9c33-615768795c1d)

How to do that?

**Comment by alexvoss** [comment, marked as answer]
You could use `navigation.tabs.sticky` to turn off the tabs block on `base.html`, then edit the `header.html` partial to include the tabs in a different place. They are currently placed after the header, not within. You may also need to play with CSS to make sure the tabs render properly as part of the header. Be aware of the drawback that editing partials means that subsequent changes to them in Material might not play with your [customization](https://squidfunk.github.io/mkdocs-material/customization/).

**Reply by jcyrss** [reply]
It works, thanks! 
And I also need to modify the style of nav tab from to `width:100%` to `width:auto`, otherwise `md-header__topic` will be covered.

So, better setting location of nav tab could be a built-in feature.

**Reply by alexvoss** [reply]
There are a million things that could be built-in but that would make the codebase and the documentation explode and make maintenance impossible. The aim instead is to make it easy to adapt things to specific needs.
